tests/Features/Properties/ModelPropertyExtension.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
Would be cool have more tests here.

Reply:
What kind of tests? I couldn't think of anything different. There are tests mainly for normal return types, date casts (Carbon) and casting according to `$casts` property of the model.

I didn't want to write a test for each type (integer, text, json etc.) Cause mainly it's the same code.

Also outside of these cases, I tested the whole branch in many different Laravel projects.